client/container.go

```diff
@@ -772,7 +746,7 @@ type ResAnnounceSpace struct {
 //
 // Return statuses:
 //   - global (see Client docs).
```

Comment:
No longer relevant. Or maybe just should reference `apistatus`.

Reply:
removed